### PR TITLE
feat(chat): add possibility to preview shared prompt (Issue #743)

### PR DESCRIPTION
**Pull request by Alexander-Kezik**
- Issue: https://github.com/epam/ai-dial-chat/issues/743

### CODE REVIEW

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-806.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Common/ItemContextMenu.tsx [hunk 1]
```diff
@@ -119,6 +122,13 @@ export default function ItemContextMenu({
         Icon: IconCopy,
         onClick: onDuplicate,
       },
+      {
+        name: t('View'),
+        display: !!onView && isExternal,
+        dataQa: 'view',
+        Icon: IconEye,
+        onClick: onView,
```
Comment (IlyaBondar) [inline apps/chat/src/components/Common/ItemContextMenu.tsx:130]: React Hook useMemo has a missing dependency: 'onView'. Either include it or remove the dependency array

**Review by IlyaBondar — CHANGES_REQUESTED** [review]

File: apps/chat/src/components/Promptbar/components/PromptModal.tsx [hunk 2]
```diff
@@ -214,7 +214,7 @@ export const PromptModal: FC<Props> = ({ isOpen, onClose, onUpdatePrompt }) => {
             <textarea
               ref={descriptionInputRef}
               name="description"
-              className={inputClassName}
+              className={classNames(inputClassName)}
```
Comment (IlyaBondar) [inline apps/chat/src/components/Promptbar/components/PromptModal.tsx]: I suppose we can remove `classNames` here

File: apps/chat/src/components/Promptbar/components/PromptModal.tsx [hunk 3]
```diff
@@ -172,7 +172,7 @@ export const PromptModal: FC<Props> = ({ isOpen, onClose, onUpdatePrompt }) => {
             : ModalState.OPENED
           : ModalState.CLOSED
       }
-      heading={t('Edit prompt')}
+      heading={`${t('Edit prompt')}: ${name}`}
```
Comment (IlyaBondar) [inline apps/chat/src/components/Promptbar/components/PromptModal.tsx]: Dio we really need it? `name` is empty initially

File: apps/chat/src/components/Promptbar/components/PromptModal.tsx [hunk 4]
```diff
@@ -246,7 +246,7 @@ export const PromptModal: FC<Props> = ({ isOpen, onClose, onUpdatePrompt }) => {
               data-qa="prompt-value"
             />
           </div>
-          <div className="flex justify-end">
+          <div className={'flex justify-end'}>
```
Comment (IlyaBondar) [inline apps/chat/src/components/Promptbar/components/PromptModal.tsx]: revert it, please

**Review by IlyaBondar — DISMISSED** [review]

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-806.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Promptbar/components/PreviewPromptModal.tsx [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+                }}
+                className="flex cursor-pointer items-center justify-center rounded p-[5px] hover:bg-accent-tertiary-alpha hover:text-accent-tertiary"
+              >
+                <IconFileArrowRight size={24} strokeWidth="1.5" />
+              </button>
+              <button
+                onClick={onUnshare}
+                className="flex cursor-pointer items-center justify-center rounded p-[5px] hover:bg-accent-tertiary-alpha hover:text-accent-tertiary"
+              >
+                <IconTrashX size={24} strokeWidth="1.5" />
```
Comment (IlyaBondar) [inline apps/chat/src/components/Promptbar/components/PreviewPromptModal.tsx]: need to add tooltips for these two buttons

**Review by IlyaBondar — APPROVED** [review]

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-806.nightly-test.deltixhub.io
>E2E tests status: success

**Pull request merged**